src/printer/index.js

```diff
@@ -143,13 +143,13 @@ function printIf(path, print) {
     const n = path.getValue();
 
     if (n.orelse && n.orelse.length > 0) {
-      if (n.orelse[0].ast_type === "If") {
+      if (n.orelse.length === 1 && n.orelse[0].ast_type === "If") {
         return concat(path.map(printElseBody, "orelse"));
       }
       return concat([
         hardline,
         "else:",
-        concat(path.map(printElseBody, "orelse"))
+        indent(concat(path.map(printElseBody, "orelse")))
```

Comment:
Just don't use `printElseBody` here and you should be good. You only need special-case handling for the `elif` case.